search_action_domain/data/ir_actions_server_data.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="create_as_ir_filter" model="ir.actions.server">
+        <field name="name">Create As Filter</field>
+        <field name="model_id" ref="base.model_ir_actions_server"/>
+        <field name="binding_model_id" ref="base.model_ir_actions_server"/>
+        <field name="binding_type">action</field>
+        <field name="state">code</field>
+        <field name="code">
+for record in records:
```

Comment:
- Move to Python code + a button
- Use `_update_xmlid`
- Use the module name in the external ID, instead of `__export__` (making sure to use it noupdate). This will need a migration script

Reply:
Done!

Reply:
> Move to Python code + a button

Done, thanks.


> Use the module name in the external ID, instead of `__export__` (making sure to use it noupdate). This will need a migration script

- Here you used the model, it should be the module. Remember external IDs are in the form `module.record_name`. The reason to use this module as module is to know exactly what records were created by the feature and for the system to being able to remove them if the module gets uninstalled.
- Migration script is pending

> Use `_update_xmlid`

Sorry, I just noticed I wrote the wrong method, I meant `_load_records` ([here](https://git.vauxoo.com/vauxoo/sbdgroup/-/blob/db74d5e1527d57b48e94e694e83610b65867ec2e/sbd_importer/models/sbd_importer.py#L266) an example). That method:
- can create a record with its external ID
- Can receive several records at once
- You don't need to validate if it already exists. If it doesn't, it creates it. If it does, record is updated.

It's the same method internally called when loading data.